### DISCUSSION TITLE
Find the jobs configured directly on the Cronicle machine

### DISCUSSION THREAD

**Discussion by leonardoluan** [jhuckaby/Cronicle#478] [Q&A] [answered]
I'm currently getting a 500 error when trying to authenticate to Chronicle (web). I'm going to take the opportunity to redo the machine and I would like to see which jobs are registered and their respective URLS using ssh.
How can I do this?

**Comment by jhuckaby** [comment, marked as answer]
```
/opt/cronicle/bin/storage-cli.js list_get global/schedule
```
